custom_components/ha-glocaltokens/api.py

```diff
@@ -85,15 +87,16 @@ def get_alarms_and_timers_from(self, device, endpoint):
 
     def get_google_devices_information(self):
         _LOGGER.debug("Fetching data...")
-        devices = self._client.get_google_devices_json()
+        devices = json.loads(self._client.get_google_devices_json())
```

Comment:
You don't necessarily need to fetch `json` and load it here. It would be more convenient to for us to just call `get_google_devices` (See [source](https://github.com/leikoilja/glocaltokens/blob/1719fde7e1f187dd05aafda6e4b074c5285ffcd6/glocaltokens/client.py#L206)) method, that will return the list back.

Json method is nice and i believe is important to keep for other use cases/packages.